packages/wallets/src/features/cashier/modules/WithdrawalVerification/WithdrawalVerificationSent/__tests__/WithdrawalVerificationSent.spec.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import WithdrawalVerificationSent from '../WithdrawalVerificationSent';
+
+describe('WithdrawalVerificationSent', () => {
+    jest.useFakeTimers();
+
+    test('should render component correctly', () => {
+        render(<WithdrawalVerificationSent counter={0} sendEmail={jest.fn()} />);
+
+        expect(screen.getByText("We've sent you an email.")).toBeInTheDocument();
+
+        expect(
+            screen.getByText('Please check your email for the verification link to complete the process.')
+        ).toBeInTheDocument();
+
+        const emailSentIcon = screen.getByTestId('dt_withdrawal_verification_sent_icon');
+        expect(emailSentIcon).toBeInTheDocument();
+
+        expect(screen.getByRole('button', { name: "Didn't receive the email?" })).toBeInTheDocument();
+    });
+
+    test('should call sendEmail function and toggle showResend on button click', async () => {
```

Comment:
showResend is internval variable of that component, so typically its better to derive name from the actuall feature which is tested, e.g. "calls sendEmail prop and shows resend email button" 
basically, we're interested in input/output of compononent, internal variables ideally should be irrelevant (with some minor exceptions)

Reply:
Updated